### DISCUSSION TITLE
How to use volar/services in v2?

### DISCUSSION THREAD

**Discussion by kingyue737** [vuejs/language-tools#4290] [Q&A]
I just noticed that `volar.config.js` is no longer supported since v2 which was mentioned in the "refactor" section of the Changelog.
Then how to use the services in https://github.com/volarjs/services? Are they all deprecated?

**Comment by ilyarefraction** [comment]
I'm also interested in this discussion. Previously we shipped vue takeover mode support through `volar.config.js` in our [extension](https://github.com/zardoy/typescript-vscode-plugins/blob/develop/typescript/src/volarConfig.ts#L120-L133). 

Can someone provide a hint or an example how to use services in v2?
